source/common/json/json_internal.cc

```diff
@@ -26,6 +26,10 @@ namespace Nlohmann {
 
 namespace {
 
+// Nlohmann JSON library will replace the invalid UTF-8 characters with this if the
+// `replace` handler is set.
```

Comment:
this specific sequence is intrinsic to the nlohmann library? Is there a doc reference at least for this, that you could put in the comment?